sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SetupAccountActivity.kt

```diff
@@ -124,8 +159,29 @@ class SetupAccountActivity : DuckDuckGoActivity(), SetupFlowListener {
     companion object {
         private const val TAG_ENABLE_SYNC = "tag_enable_sync"
         private const val TAG_RECOVER_ACCOUNT = "tag_recover_account"
-        fun intent(context: Context): Intent {
-            return Intent(context, SetupAccountActivity::class.java)
+        private const val TAG_DEVICE_CONNECTED = "tag_device_connected"
+
+        enum class Screen {
+            SETUP,
+            RECOVERY_CODE,
+            DEVICE_CONNECTED,
+        }
+
+        const val SETUP_ACCOUNT_SCREEN_EXTRA = "SETUP_ACCOUNT_SCREEN_EXTRA"
+
+        fun intentStartSetupFlow(context: Context): Intent {
```

Comment:
you can get rid of this method by adding a default parameter to `fun intent()`

```
fun intent(
      context: Context,
      screen: Screen = SETUP,
  ): Intent {
      return Intent(context, SetupAccountActivity::class.java).apply {
          putExtra(SETUP_ACCOUNT_SCREEN_EXTRA, screen)
      }
  }
```

Reply:
I thin I'm going to keep the method as it its, and rename the other to be explicit about what step will navigate to. I prefer to remove `Screen` so callers don't know about that.